core/src/main/java/de/muenchen/allg/itd51/wollmux/func/Values.java

```diff
@@ -28,37 +28,36 @@
 import java.util.Map.Entry;
 
 /**
- * Eine Menge benannter {@link de.muenchen.allg.itd51.wollmux.func.Value}s.
+ * Plenty of known {@link de.muenchen.allg.itd51.wollmux.func.Value}s.
  *
  * @author Matthias Benkmann (D-III-ITD 5.1)
  */
 public interface Values
 {
   /**
-   * Liefert true genau dann wenn ein Wert mit der ID id vorhanden ist (ACHTUNG, bei
-   * mit BIND zusammengesetzten Funktionen bekommt die gebundene Funktion unter
-   * Umständen hier keine akkurate Antwort).
+   * Deliver exact true then, if a value with the ID id exists (WARNING, the bound
```

Comment:
This is not proper english, please improve the grammar (check deepl.com for reference).